kiwi/exceptions.py

```diff
@@ -131,6 +131,10 @@ class KiwiDataStructureError(KiwiError):
     pass
 
 
+class KiwiDescriptionConflict(KiwiError):
+    pass
```

Comment:
I agree, I will add doc strings explaining the exceptions in an extra pull request ok ?